pkg/api/parser/text/text_test.go

```diff
@@ -0,0 +1,154 @@
+package text
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/prometheus/prometheus/pkg/timestamp"
+	"github.com/timescale/promscale/pkg/pgmodel/ingestor"
+	"github.com/timescale/promscale/pkg/pgmodel/model"
+	"github.com/timescale/promscale/pkg/prompb"
+)
+
+func TestDBIngestorParseTextData(t *testing.T) {
+	defaultScrapeTime := time.Now()
+
+	timeProvider = func() time.Time {
+		return defaultScrapeTime
+	}
+
+	testCases := []struct {
```

Comment:
please add 1 test cases:
- multiple metrics with at least one metric with multiple series in one input string